docs/docs/specs/banking/history.md

```diff
@@ -64,11 +64,18 @@ query {
 
 Always provide a method for users to **download their transaction history**.
 
-The source code provides this with button in the history tab.
 When a user clicks **Account statements**, a pop-in appears with a list of months for which downloadable account statements are available.
 
```

Comment:
@bloodyowl I would add the information here, actually, and in an unordered list. We should also update the screenshot.

- Monthly: Account statements are generated automatically at the end of each month. They're available in PDF format only and in the account's default language.
- Custom: Generate your own account statements for a custom date range (up to 3 months). You can choose PDF or CSV format and any of Swan's supported languages.